uvtools/dspec.py

```diff
@@ -837,7 +837,7 @@ def _get_bl_dly(bl_len, horizon=1., standoff=0., min_dly=0.):
     return bl_dly
 
 
-def gen_window(window, N, alpha=0.5, edgecut_low=0, edgecut_hi=0, **kwargs):
+def gen_window(window, N, alpha=0.5, edgecut_low=0, edgecut_hi=0, normalization='none', **kwargs):
```

Comment:
I think it's a little cleaner to set the default argument to normalization be the python keyword `None` rather than a string, but this also works.

Reply:
I think None does make things clearer (also see below). I've changed this.